kinto/__main__.py

```diff
@@ -34,6 +37,10 @@ def main(args=None):
 
     args = vars(parser.parse_args())
     config_file = args['ini_file']
+
+    if not os.path.exists(config_file):
+        shutil.copy('config/kinto.ini.sample', config_file)
```

Comment:
Shall we do that in the init command instead? and tell the person that she need to start ``kinto init`` first?

In the make file we can also add the CONFIG_FILE as a dependency and run ``kinto init`` automatically if the file doesn't exists.